src/content/public-goods-funding/index.md

```diff
@@ -0,0 +1,289 @@
+
+#### Principal-agent problem {#principal-agent-problem}
+
+The [principal-agent problem](https://en.m.wikipedia.org/wiki/Principal%E2%80%93agent_problem) is a concept that explains the conflict of interests between an individual (the principal) and another entity acting on their behalf (the agent). While the agent is expected to act in the best interests of their principal, they may pursue self-seeking ends.
+
+In the context of public goods, the principal-agent problem concerns differences in opinion between a group (e.g., citizens) and their representatives (e.g., elected government officials) as it concerns the allocation of resources to public goods. In this case, what the group considers a public good may be different from what representatives consider a public good.
+
+The problem is further compounded by lack of transparency and [information asymmetries](https://en.m.wikipedia.org/wiki/Information_asymmetry). Often, those endowed with power to allocate resources to public goods misuse it and prevent accountability by shrouding administrative processes in secrecy.
+
+### 2. Poor resource-allocation mechanisms {#poor-resource-allocation-mechanisms}
+
+In funding public goods, it is necessary to have a mechanism for eliciting the preferences of individuals before allocating funds. Traditional public goods funding models are inefficient because they rely on poorly designed mechanisms for measuring public preferences, such as:
+
+#### Democratic voting {#democratic-voting}
+
+Funding models that apply democratic voting ("one person, one vote") rules rely on the majority opinion to allocate funding for public goods. In theory, using the collective opinion to guide funding is a great idea. In reality, however, this system has problems:
+
+1. **Rational indifference**: In a democratic voting scheme, the ability of each individual to influence the outcome is limited (you can only vote once). As such, those whom a public good benefits the most (for example, public healthcare for aged citizens) are unable to push for its production. This leads to individuals rationalizing nonparticipation ("Why bother voting when my vote matters little?").
+
+2. **Voter ignorance**: Prioritizing the preferences of the majority may not necessarily lead to spending on the right public goods owing to widespread voter ignorance. In this case, informed voters are unable to push for the production of more beneficial public goods.
+
+3. **Negative externalities**: Another downside with democratic funding schemes is the cost borne by the minority. In democratic systems, those who may not necessarily need a public good are forced to pay the same as those who need it the most. For example, Bob (a risk-averse individual) is forced to pay taxes to subsidize medical procedures for Alice (a risk-seeking individual), even though Bob himself might never need such services.
```

Comment:
I don't like this example. Public health is a great public good, and we all use it even if it's later in life.

Reply:
I just replaced with a less controversial example. Let me know if it works.